globals/colours.css

```diff
@@ -13,9 +15,12 @@
   --color-brown:        rgb(156,120,102);
   --color-brown-dark:   rgb(96,79,76);
 
+  --color-blue-muted:   rgb(225,235,242);
```

Comment:
They're used on the careers page. In future it might be worthwhile to put together a _theme_ of colours like `--theme-blue-fg`, `--theme-blue-bg`...